Audio Service/ServiceLocator.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using Medrick.Unolit.Utility;
+
+namespace Medrick.Unolit.Service
+{
+    public class ServiceLocator : Singleton<ServiceLocator>
```

Comment:
this class should not be in the AudioService folder, it's in the ServiceLocator already